### PR TITLE
Improves what-is-maven.md

### PR DESCRIPTION
**Pull request by manaswinidas**
Removes grammatical errors

### CODE REVIEW

File: content/markdown/what-is-maven.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ navigate any other project that uses Maven and the same defaults.
 ### Allowing transparent migration to new features
 
 Maven provides an easy way for Maven clients to update their
-installations so that they can take advantage of any changes that been
+installations so that they can take advantage of any changes that has been
```
Comment (pzygielo) [inline content/markdown/what-is-maven.md]: I'm no native speaker, but for me added `has` does not agree with plural `changes`.

Comment (michael-o) [reply content/markdown/what-is-maven.md]: Indeed, I missed that one too.

Comment (manaswinidas) [reply content/markdown/what-is-maven.md]: @michael-o @pzygielo should I create a new PR? I don't think I can make changes here.

Comment (michael-o) [reply content/markdown/what-is-maven.md]: @manaswinidas Please open a new one, thanks!

**Comment by jira-importer** [comment]
Resolve #1095

**Pull request merged**